README.md

```diff
@@ -56,6 +56,12 @@ make
 you've installed kiwix-lib (and other libraries) you may have to
 update the env variable `PKG_CONFIG_PATH`.
 
+It can be set as follows, for example, for x86-64 native systems
```

Comment:
Just put that line behind the . (without a carriage return), to "stick" things together in the same paragraph.

Reply:
@kelson42 done.